bot/seasons/halloween/8ball.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'halloween', 'responses.json'), 'r', encoding="utf8") as f:
+    responses = json.load(f)
+
+
+class EightBall(commands.Cog):
+    """Spooky Eightball answers."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=('spooky8ball',))
+    async def spookyeightball(self, ctx, *, question: str):
+        """Responds with a random response to a question."""
+        choice = random.choice(responses['responses'])
+        if len(choice) == 1:
+            await ctx.send(choice[0])
+        else:
```

Comment:
The difference in behavior between the single response sending a plantext message and the multi-response sending an embed is quite drastic. Let's make this consistent (and more compact) and just edit `msg.content` with the updated string.